android/src/main/java/com/sbugert/rnadmob/RNPublisherBannerViewManager.java

```diff
@@ -242,8 +244,17 @@ public void addView(ReactPublisherAdView parent, View child, int index) {
 
     @ReactProp(name = PROP_AD_SIZE)
     public void setPropAdSize(final ReactPublisherAdView view, final String sizeString) {
-        AdSize adSize = getAdSizeFromString(sizeString);
-        view.setAdSize(adSize);
+        AdSize adSize;
+        if(isNumeric(sizeString)){
+            adSize = new AdSize(Integer.parseInt(sizeString), nativeAdHeight);;
```

Comment:
Extra semi-column here. Also thinking we should pass in the WxH as the size param but this will work for the current release.